scraper.go

```diff
@@ -169,6 +170,28 @@ func NewScraper(hostname, statefile, metalogfile string) *Scraper {
 	return s
 }
 
+func (s *Scraper) LoadConfiguration(file string) {
+	config, err := LoadLogScraperConfig(file)
+	if err != nil {
+		fmt.Print(err)
+		s.logMetaf("Error opening configuraton file: %v", err)
+		return
+	}
+
+	logSources, errors := config.LogSources()
+	if errors != nil && len(errors) > 0 {
+		for _, err := range errors {
+			s.logMetaf("Error parsing configuraton file: %v", err)
+		}
+		return
+	}
+
+	s.Sources = append(s.Sources, logSources...)
+	for _, src := range s.Sources {
+		fmt.Printf("Source loaded: %v\n", src)
```

Comment:
I dont quite understand, this is just printing out each source for debug purposes